packages/ui/src/components/modal/modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC, ReactNode } from "react";
```

Comment:
personally I'd prefer using [headless](https://headlessui.dev/react/dialog) since we already use it, though this is fine for now